client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/dynamic-settings-form.js

```diff
@@ -170,6 +179,7 @@ const mapDispatchToProps = ( dispatch ) => ({
 DynamicCarrierAccountSettingsForm.propTypes = {
 	carrierType: PropTypes.string,
 	carrierName: PropTypes.string,
+	carrierDesc: PropTypes.string,
```

Comment:
What are your thoughts on calling the prop `carrierDescription`? (without abbreviations)
There are some abbreviations that are common in codebases (`err`, `res`, `req`), but all the other props are not abbreviated

Reply:
Done!